projects/1.21/assets/occultism/occultism/lang/zh_cn.json

```
@@ -0,0 +1,3191 @@
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_otherworld_bird.description2.text": "要获取用于献祭的鹦鹉或鹦鹉使魔，可以考虑通过[野生鹦鹉仪式](entry://possession_rituals/possess_unbound_parrot)或[鹦鹉使魔仪式](entry://familiar_rituals/familiar_parrot)进行召唤。\n\\\n\\\n**提示**：若使用了防止宠物死亡的模组，请选择野生鹦鹉仪式！\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_otherworld_bird.entity.text": "**提供**：[#](ad03fc)多段跳[#]()，[#](ad03fc)跳跃提升[#]()，[#](ad03fc)缓降[#]()\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_otherworld_bird.name": "具翼使魔",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_parrot.description.text": "仪式中，一只[#](ad03fc)魔精[#]()会被**作为使魔**召唤，而对[#](ad03fc)鸡[#]()的屠宰和供奉的染料能诱使[#](ad03fc)魔精[#]()选用鹦鹉的形态。\\\n不过，由于[#](ad03fc)魔精[#]()并不属于最聪明的魔灵之列，它们有时会误解这一指示……\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_parrot.description.title": "描述",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_parrot.description2.text": "*也就是说，要是生成了一只[#](ad03fc)鸡[#]()的话，这不是Bug，是单纯的运气不好！*\n\\\n\\\n**升级效果**\\\n不能被铁匠使魔升级。\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_parrot.entity.text": "**提供**：[#](ad03fc)陪伴[#]()\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_parrot.name": "鹦鹉使魔",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_shub_niggurath.description.text": "**升级效果**\\\n被铁匠使魔升级后，莎布·尼古拉丝使魔会获得一个警钟。每当你打中该使魔时，它就会敲响警钟，吸引大范围的敌人。\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_shub_niggurath.description.title": "描述",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_shub_niggurath.entity.text": "**提供**：[#](ad03fc)生成小号的自己为你战斗[#]()\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_shub_niggurath.name": "莎布·尼古拉丝使魔",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_shub_niggurath.ritual.text": "[#](ad03fc)莎布·尼古拉丝[#]()并不能直接召唤。首先，你需要召唤一只[奇美拉使魔](entry://familiar_rituals/familiar_chimera)，并通过对其喂食[](item://minecraft:golden_apple)来使[#](ad03fc)山羊[#]()分离。将这只山羊带到一个[#](ad03fc)森林群系[#]()。然后按顺序对山羊使用[任意黑色染料](item://minecraft:black_dye)、[](item://minecraft:flint)和[](item://minecraft:ender_eye)就能召唤[#](ad03fc)莎布·尼古拉丝[#]()。\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.familiar_shub_niggurath.ritual.title": "仪式",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.iesnium_golem.description": "无形傀儡",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.iesnium_golem.description.text": "艾瑟金属傀儡是普通铁傀儡的直接升级版，这个新版本更加强大且无敌。\\\n只有玩家可以通过潜行状态下攻击来解除它们，解除后会返还对应的[](item://occultism:fragile_soul_gem)。\n",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.iesnium_golem.description.title": "艾瑟金属傀儡",
+  "book.occultism.dictionary_of_spirits.familiar_rituals.iesnium_golem.familiar.text": "由于相似性，在此书中艾瑟金属傀儡章节是使魔页面的一部分，不过显然这种傀儡并不是使魔，因为它没有主人。\n\\\n\\\n它们并不能被存储在[](item://occultism:familiar_ring)中。然而你还是可以使用[](item://occultism:soul_gem)像存储其他生物一样存储它们。\n",
```

```suggestion
  "book.occultism.dictionary_of_spirits.familiar_rituals.iesnium_golem.familiar.text": "由于相似性，在此书中艾瑟金属傀儡章节是使魔页面的一部分，不过显然这种傀儡并不是使魔，因为它没有主人。\n\\\n\\\n它们并不能被存储在[](item://occultism:familiar_ring)或[](item://occultism:soul_gem)中。然而你还是可以使用像前页所述方法解除它们。\n",
```
`The Iesnium Golem Chapter is part of the Familiar Category of this book because of the similarities, however a golem is not exactly a familiar since it does not have an owner.\n\\\n\\\nAs such, they also cannot be stored in a [](item://occultism:familiar_ring) or in a [](item://occultism:soul_gem). You can, however, dispense with them as stated on the previous page.\n`
不太清楚这里的dispense with是指什么，不过前页是说怎么拆，就先这么发suggestion了